components/CategoryPage.js

```diff
@@ -65,13 +67,26 @@ var CategoryList = React.createClass({
 var Category = React.createClass({
   render: function() {
     return  (
-      <li className="category_list_item">
-        <div className="category-name">
+    <div>
+      <div className="col-lg-2 col-md-2 col-sm-2 hidden-xs category_option">
```

Comment:
The bootstrap classes scale up. So if you had only used the class col-xs-8, it would use 8 columns for all screen sizes. As such, your classnames can be simplified here into "col-xs-1 col-md-2".